### PR TITLE
travis-ci integration for Development branch, generation script for readme

### PR DESCRIPTION
**Pull request by kurt-hectic**
@LuisaIckes , @efucile , @joergklausen , @marqh ,

this PR integrates the travis-ci also into the development branch.
It also has a new script which can generate the readme.md from the wmdr-tables.csv file.
The travis-ci script actually runs this script to check if readme.md and wmdr-tables.csv are in sync.  The routine also checks if all CSV files in the directory tables_en are referenced in wmdr-tables.csv and vice versa.

In short.. if you have added a new codelist you need to update wmdr-tables.csv and then run ```python scripts/generate-readme.py``` which will update the readme.md file.

### CODE REVIEW

**Comment by joergklausen** [comment]
Hi Timo
Looks good, thanks!
Jörg

Sent from my iPhone

On 4 Nov 2019, at 17:43, Timo Pröscholdt <notifications@github.com> wrote:

﻿

@LuisaIckes<https://github.com/LuisaIckes> , @efucile<https://github.com/efucile> , @joergklausen<https://github.com/joergklausen> , @marqh<https://github.com/marqh> ,

this PR integrates the travis-ci also into the development branch.
It also has a new script which can generate the readme.md from the wmdr-tables.csv file.
The travis-ci script actually runs this script to check if readme.md and wmdr-tables.csv are in sync. The routine also checks if all CSV files in the directory tables_en are references in wmdr-tables.csv and vice versa.

In short.. if you have added a new codelist you need to update wmdr-tables.csv and then run python scripts/generate-readme.py > tables_en/readme.me

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/wmo-im/wmds/pull/139

Commit Summary

  *   added travis and test scripts, new file for codetable metadata
  *   added python to script command
  *   corrected registry URLs
  *   new generation script for readme and integration into travis-ci script
  *   new generation script for readme and integration into travis-ci script
  *   new readme
  *   corrected spelling in 5-02-05

File Changes

  *   A .gitignore<https://github.com/wmo-im/wmds/pull/139/files#diff-0> (2)
  *   A .travis.yml<https://github.com/wmo-im/wmds/pull/139/files#diff-1> (10)
  *   A scripts/generate-readme.py<https://github.com/wmo-im/wmds/pull/139/files#diff-2> (4)
  *   A scripts/registry_deployment/__init__.py<https://github.com/wmo-im/wmds/pull/139/files#diff-3> (2)
  *   A scripts/registry_deployment/generator.py<https://github.com/wmo-im/wmds/pull/139/files#diff-4> (266)
  *   A scripts/registry_deployment/registry.py<https://github.com/wmo-im/wmds/pull/139/files#diff-5> (84)
  *   A scripts/requirements_deployment.txt<https://github.com/wmo-im/wmds/pull/139/files#diff-6> (1)
  *   A scripts/test-gen.py<https://github.com/wmo-im/wmds/pull/139/files#diff-7> (34)
  *   M tables_en/readme.md<https://github.com/wmo-im/wmds/pull/139/files#diff-8> (113)
  *   A tables_en/wmdr-tables.csv<https://github.com/wmo-im/wmds/pull/139/files#diff-9> (55)

Patch Links:

  *   https://github.com/wmo-im/wmds/pull/139.patch
  *   https://github.com/wmo-im/wmds/pull/139.diff

**Comment by kurt-hectic** [comment]
@efucile .. please accept the PR.. we need the script to keep the readme synced
